### DISCUSSION TITLE
Remove ObjectEntryPage

### DISCUSSION THREAD

**Discussion by kistenklaus** [mu-zero-HYPERLOOP/control-panel#106] [Ideas]
The Object Entry Page feels weird to navigate to with expanding by clicking on the small icon. 

What do you think about removing the object entry page completely and having the graphs as a [MaterialUI Modal](https://mui.com/material-ui/react-modal/) that appears when you press the graph icon (right now it's a link to the object entry page)
on the node page. 

Not sure but i feel like this might make navigation a bit easier especially if we also slightly increase the size of the text in the 
sidebar.

This also avoids to long object entry names in the sidebar. 

What do you think?

@arrowtip @Nico4899 @NiklasHeneka

**Comment by NiklasHeneka** [comment]
Depends on what else we want to display on the Object Entry page. We talked about displaying things like the bitrate or how up-to-date this value is or where this sensor is located in the pod. 

So if we want to display more than just graphs (if we find something cool to display about OE) we could keep it. Otherwise we can just drop the page, it's fine for me.
@kistenklaus
